test/data/repositories/rhel-8.json

```diff
@@ -1,55 +1,66 @@
-    },
-    {
-      "name": "rt",
```

Comment:
This patch removes the definition for the RT repository. IIRC we still need this, right ?